### PR TITLE
Fix example export dbadbrole

### PR DESCRIPTION
**Pull request by Splaxi**
## Type of Change

 - [X] Documentation
 

### Purpose
Update example documentation based on my automated example testing script

Someone might need to spend some quality time with the examples and their description, to make sure that they reflect reality...

### CODE REVIEW

**Comment by wsmelton** [comment]
> Someone might need to spend some quality time with the examples and their description, to make sure that they reflect reality...

If you would like to fix the examples please do so in this PR and then we can accept it.

File: functions/Export-DbaDbRole.ps1 [hunk 1]
```diff
@@ -97,40 +97,40 @@ function Export-DbaDbRole {
...
     .EXAMPLE
-        PS C:\> Export-DbaDbRole -SqlInstance sqlserver2008 -Login realcajun -Path C:\temp\users.sql
+        PS C:\> Export-DbaDbRole -SqlInstance sqlserver2008 -Role realcajun -Path C:\temp\roles.sql
 
-        Exports login realcajun from sqlserver2008 to the file C:\temp\users.sql
+        Exports role realcajun from sqlserver2008 to the file C:\temp\roles.sql
 
     .EXAMPLE
-        PS C:\> Get-DbaDatabase -SqlInstance sqlserver2008 -Login realcajun | Export-DbaDbRole
+        PS C:\> Get-DbaDatabase -SqlInstance sqlserver2008 | Export-DbaDbRole -Role realcajun
```
Comment (wsmelton) [inline functions/Export-DbaDbRole.ps1]: Example does not depict what the description is....

File: functions/Export-DbaDbRole.ps1 [hunk 2]
```diff
@@ -97,40 +97,40 @@ function Export-DbaDbRole {
     .EXAMPLE
         PS C:\> Export-DbaDbRole -SqlInstance sql2005 -Path C:\temp
 
-        Exports the Database Roles for all  for SQL Server "sql2005" and writes them to the file "C:\temp\sql2005-logins.sql"
+        Exports the Database Roles for SQL Server "sql2005" and writes them to the file "C:\temp\sql2005-logins.sql"
```
Comment (wsmelton) [inline functions/Export-DbaDbRole.ps1]: This should say "exports all the database roles for..."

File: functions/Export-DbaDbRole.ps1 [hunk 3]
```diff
@@ -97,40 +97,40 @@ function Export-DbaDbRole {
...
 
-        Exports ONLY logins FROM sqlserver2014a with permissions on databases HR and Accounting
+        Exports ONLY roles FROM sqlserver2014a with permissions on databases HR and Accounting
 
     .EXAMPLE
-        PS C:\> Export-DbaDbRole -SqlInstance sqlserver2008 -Login realcajun, netnerds -Path C:\temp\login.sql -ExcludeGoBatchSeparator
+        PS C:\> Export-DbaDbRole -SqlInstance sqlserver2008 -Role realcajun,netnerds -Path C:\temp\roles.sql
 
-        Exports ONLY logins netnerds and realcajun FROM sqlserver2008 server, to the C:\temp\login.sql file without the 'GO' batch separator.
+        Exports ONLY roles netnerds and realcajun FROM sqlserver2008 server, to the C:\temp\roles.sql file.
```
Comment (wsmelton) [inline functions/Export-DbaDbRole.ps1]: The way this example was updated it is a duplicate of the example 3. This should be updated to show use of the `-BatchSeparator` parameter.

Comment (Splaxi) [reply functions/Export-DbaDbRole.ps1]: Would you prefer:

```
PS C:\> Export-DbaDbRole -SqlInstance sqlserver2008 -Role realcajun,netnerds -Path C:\temp\roles.sql -BatchSeparator "GO"
```

or

```
PS C:\> Set-DbatoolsConfig -FullName formatting.batchseparator -Value "GO"
PS C:\> Export-DbaDbRole -SqlInstance sqlserver2008 -Role realcajun,netnerds -Path C:\temp\roles.sql
```

The `-BatchSeparator` is defaulted with the following: 
```
...
[string]$BatchSeparator = (Get-DbatoolsConfigValue -FullName 'Formatting.BatchSeparator'),
...
```

Comment (wsmelton) [reply functions/Export-DbaDbRole.ps1]: The default of `formatting.batchseperator` is `GO` already. The description of the example stated `without the 'GO' batch separator`. I don't have test setup to run but that would either be setting `-BatchSeparator $null` or using `Set-DbatoolsConfig` to set the configuration value to `$null`.

Comment (Splaxi) [reply functions/Export-DbaDbRole.ps1]: Will include both! Thanks for the feedback.

**Pull request merged**